readthedocs/projects/tasks.py

```diff
@@ -1146,6 +1150,56 @@ def fileify(version_pk, commit):
         )
 
 
+def _update_intersphinx_data(version, path, commit):
```

Comment:
Why not to run this function in a celery tasks instead of calling syncly?

Reply:
It is already run inside of a celery task.